data.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy.sql import text
+from models import db
+
+
+def statewide_by_day(current_days_before):
+    sql = """
+        SELECT * FROM cumulative_stats_by_day
+        WHERE days_before < 45 AND days_before > :current_days_before;
```

Comment:
if you want, this could conceivably be something like:
`days_before > (select '2021-01-05' - max(date(file_update_time)) from updated_times)`
(possible i have a fencepost error there, please double-check my math)